app/src/test/java/mollie/tictactoe/AndroidPlayerFactoryTest.java

```diff
@@ -0,0 +1,18 @@
+package mollie.tictactoe;
+
+import org.junit.Test;
+
+import ttt.Player;
+
+import static junit.framework.Assert.assertEquals;
+import static mollie.tictactoe.AndroidPlayerType.HUMAN;
+import static ttt.game.Marks.X;
+
+public class AndroidPlayerFactoryTest {
+
+    @Test
+    public void returnsAMobileHumanPlayer() {
```

Comment:
Here your test name mentions 'Mobile' but the classes are 'Android' as Felipe already mentioned, which is a little inconsistent. Perhaps 'MobilePlayer' or 'GuiPlayer' would be better naming for the classes.